tests/test_ffmpeg_builder.py

```diff
@@ -1,15 +1,31 @@
 from shutil import which
 
 from facefusion import ffmpeg_builder
-from facefusion.ffmpeg_builder import chain, run, select_frame_range, set_audio_quality, set_audio_sample_size, set_stream_mode, set_video_quality
+from facefusion.ffmpeg_builder import chain, concat, keep_video_alpha, run, select_frame_range, set_audio_quality, set_audio_sample_size, set_stream_mode, set_video_encoder, set_video_fps, set_video_quality
 
 
 def test_run() -> None:
 	assert run([]) == [ which('ffmpeg'), '-loglevel', 'error' ]
 
 
 def test_chain() -> None:
-	assert chain(ffmpeg_builder.set_progress()) == [ '-progress' ]
+	assert chain(ffmpeg_builder.set_progress()) == ['-progress']
```

Comment:
you changed spacing here, but I suggest this to be a better test anyways - more like test_concat